### ISSUE TITLE
Create Collective Flow: Final project polishing

### ISSUE THREAD

**Issue by sbinlondon** [opencollective/opencollective#2945]
Part of #2872 

A place to collect all _final_ final polishing and feedback for the project before launch.

## CCF Form

* Github repo card component CSS
* Think about `componentDidMount`/`componentDidUpdate` in main `NewCreateCollective` component and how this can be improved
* Navbar/Collective breadcrumb issue
* Remove heights from any buttons

## Financial Contributions

**Comment by Betree** [comment]
Something else that hasn't been addressed from https://github.com/opencollective/opencollective/issues/2937, not critical but would be nice:
- I get a loading spinner when I open https://staging.opencollective.com/create/v2. Showing the page with disabled buttons during SSR would be nice.

---

- Also we should not set any height on buttons. Using the standards from the design system (small, medium, large) has a lot of benefits. It's up to design to respect the standards from the design system, or to explicitly state when a component is changing from the norm. In this specific case, we should not change the height of the buttons just to gain the 2px that make it pixel perfect. Either our medium buttons are globally wrong and we should fix them globally, or this is a design issue.

  There's also a very concrete issue introduced by setting the height: it won't play nice with localization or with person with disabilities who set bigger font-sizes:

![image](https://user-images.githubusercontent.com/1556356/75695190-fce02d80-5ca9-11ea-937f-c09edd5343cc.png)

**TLDR ;** you can safely remove the height on the buttons, don't worry if they're not pixel perfect.

**Comment by sbinlondon** [comment]
@Betree Do you mean this loading spinner? I thought that was the desired behaviour while the page is loading?

<img width="1792" alt="Screenshot 2020-03-02 at 19 47 37" src="https://user-images.githubusercontent.com/37520401/75711708-c1039300-5cbe-11ea-94ae-9ab2c10c9824.png">

Re: button height, good catch! I had removed the height on the buttons in Collective Category Picker because @znarf had noticed it caused a bug, but forgot to remove it from these buttons too. I'll be sure to do that.

**Comment by Betree** [comment]
I meant that indeed, but after thinking a bit more about it I realize that the alternative (disabling the buttons) is not great. Feel free to ignore my suggestion about that.

**Comment by znarf** [comment]
@Betree @sbinlondon For the loading, this is a current common practice through the codebase and it's the simplest. Would be great to figure better practices for the future but that's out of scope here.

**Comment by Betree** [comment]
Some places already don't follow the pattern of showing a loading spinner tho. The idea is to acknowledge that there are three possible states, not only two:

1. User logged out, won't log in (`loadingLoggedInUser` = false, `LoggedInUser` = false)
2. User logged in (`loadingLoggedInUser` = *, `LoggedInUser` = true)
3. User logged out, loading (`loadingLoggedInUser` = true, `LoggedInUser` = *)

If done from the beginning it's actually not very complex to implement. In the case of the create expense form, it translates to:
1. User logged out, won't log in => show modal above the form to sign in
2. User logged in => show form
3. User logged out, loading => show form with disabled submit

The same could eventually apply here, but that's a minor enhancement and converting an existing page is more complex so I wouldn't push for it.

**Comment by sbinlondon** [comment]
Ideas to do:

* refactor to use `FormattedMessage` where possible
* are there GraphQL V1 queries or mutations that can be replaced with v2?
* test the flow as a whole
